src/hotspot/os/linux/cgroupV1Subsystem_linux.hpp

```diff
@@ -39,36 +39,34 @@ class CgroupV1Controller: public CgroupController {
 
     /* Constructed subsystem directory */
     char *_path;
+    void set_subsystem_path(char *cgroup_path);
```

Comment:
Come to think of it, `set_subsystem_path()` needs to remain public, since that's the hook for adjusting a controllers' path that we'd need.